src/agent/onefuzz-agent/src/tasks/stats/common.rs

```diff
@@ -21,7 +21,7 @@ pub async fn monitor_stats(path: Option<String>, format: Option<StatsFormat>) ->
                     StatsFormat::AFL => afl::read_stats(&path).await,
                 };
                 if let Ok(stats) = stats {
-                    track_event(runtime_stats, stats);
+                    event_entries!(runtime_stats; stats);
```

Comment:
previously, we did not log the AFL stats.  it was only sent to Application Insights.

Reply:
That's strange. It seems like it should have, since `track_event()` called the now-removed `local_log_event()`.